bindings/r/tests/testthat/_snaps/delimiter.md

```diff
@@ -122,7 +122,7 @@
               "]" [(1, 4), (1, 5)]
             )
           )
-          close: "]]" [(1, 5), (1, 5)]
+          close: "]]" MISSING [(1, 5), (1, 5)]
```

Comment:
We now show `MISSING` for anonymous and named nodes that have been interpolated by tree-sitter's error recovery (like they do in their cli output). This requires https://github.com/DavisVaughan/r-tree-sitter/pull/7